src/test/java/bio/terra/app/configuration/ConnectedTestConfiguration.java

```diff
@@ -11,6 +11,7 @@
 
     private String ingestbucket;
     private String ingestRequesterPaysBucket;
+    private String googleBillingAccountId;
```

Comment:
It seems like we should use resource files from the test/resources for test properties. I'm not sure why application-connectedtest.properties or integrationtest is under main.

Reply:
Hmm...that's a really good point.  I think we may want to keep them in separate files since I think it looks up the files based on runtime config but I'll try to move those into test to see what happens